Rideshare.UnitTests/Commuters/GetCommuterStatusQueryHandlerTest.cs

```diff
@@ -0,0 +1,86 @@
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using AutoMapper;
+using Moq;
+using Rideshare.Application.Common.Dtos.Security;
+using Rideshare.Application.Contracts.Identity;
+using Rideshare.Application.Features.Commuters.Queries;
+using Rideshare.Application.Responses;
+using Rideshare.Application.UnitTests.Mocks;
+using Rideshare.Domain.Models;
+using Xunit;
+
+namespace Rideshare.UnitTests.Commuters
+{
+	public class GetCommuterStatusQueryHandlerTests
+	{
+		[Fact]
```

Comment:
You should handle the dependency injection in a constructor , How are you planing to add another test in this file if you did the dependency injection in one fact